matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/auth/DefaultAuthenticationService.kt

```diff
@@ -404,6 +406,22 @@ internal class DefaultAuthenticationService @Inject constructor(
         )
     }
 
+    override suspend fun loginUsingQrLoginToken(
+            homeServerConnectionConfig: HomeServerConnectionConfig,
+            loginToken: String,
```

Comment:
Is it possible to pass all the data from the QR code here? So that the SDK can handle the next step automatically?

Reply:
At this point we don't actually have the full context yet, so it probably doesn't make sense to do this.

We can look at the overall implementation once done and see what refactoring to do.